Index: source/pug/pages/about.pug
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- source/pug/pages/about.pug	(revision )
+++ source/pug/pages/about.pug	(revision )
@@ -106,5 +106,6 @@
                                 a.contacts__link(href="#")
                                     img.contacts__item-pic(src="./assets/img/icons/map_marker.png" alt="Почта")
                                     span.contacts__item-text Москва, м. Братиславская
+        .empty
 block footer
     include ../sections/footer.pug
\ No newline at end of file
